netbox/dcim/filtersets.py

```diff
@@ -447,10 +447,14 @@ class RackReservationFilterSet(NetBoxModelFilterSet, TenancyFilterSet):
         to_field_name='username',
         label=_('User (name)'),
     )
+    unit = NumericArrayFilter(
+        field_name='units',
+        lookup_expr='contains'
+    )
```

Comment:
label?  Not completely sure these need labels, I guess the REST documentation would show it.  Noticed there are a lot missing in the filtersets.